contrib/docker/start.apache.sh

```diff
@@ -1,15 +1,39 @@
 #!/bin/bash
 
+# We should already be in /var/www, but just to be explicit
+cd /var/www || exit
+
 # Create the storage tree if needed and fix permissions
 cp -r storage.skel/* storage/
 chown -R www-data:www-data storage/ bootstrap/
 
+if [[ ! -e storage/.docker.init ]]; then
+    echo "Fresh installation, initializing database..."
+    chown www-data:www-data .env &&
+        gosu www-data php artisan key:generate &&
+        gosu www-data php artisan migrate:fresh --force &&
+        gosu www-data php artisan passport:keys &&
+        echo "done" >storage/.docker.init
+fi
+
 # Refresh the environment
-php artisan storage:link
-php artisan horizon:publish
-php artisan route:cache
-php artisan view:cache
-php artisan config:cache
-
-# Finally run Apache
-apache2-foreground
+gosu www-data php artisan storage:link
+gosu www-data php artisan horizon:publish
+gosu www-data php artisan config:cache
+# gosu www-data php artisan cache:clear
+gosu www-data php artisan route:cache
+gosu www-data php artisan view:cache
+
+# Check for migrations
+gosu www-data php artisan migrate:status | grep No && migrations=yes || migrations=no
```

Comment:
This looks like it will break easily, because we parse the output of  the migrate:status command. I mean, is it ensured, that the migration names are always lowercase, so that this grep does not falsely finds a match?
I just had a short look into https://laravel.com/docs/master/migrations. But there is no more robust way to get the migration status, or?